src/main/Domain.Model/Avatars/AvatarServer.cs

```diff
@@ -9,6 +9,7 @@ namespace ei8.Avatar.Installer.Domain.Model.Avatars
     /// </summary>
     public class AvatarServer
     {
+        public string? Id { get; set; }
```

Comment:
String is already nullable, so it's not necessary to add a question mark. Were there other reasons why it was added?

Reply:
I'll remove it right away. Thanks for the feedback. I added it based on IntelliSense's suggestion.